apps/media-viewer/src/Mediaviewer.vue

```diff
@@ -14,7 +14,7 @@
         <div class="uk-width-large uk-flex uk-flex-middle uk-flex-center uk-flex-around" style="user-select:none;">
           <oc-icon role="button" class="oc-cursor-pointer" size="medium" @click="prev" name="chevron_left" />
           <!-- @TODO: Bring back working uk-light -->
-          <span class="uk-text-small" style="color:#fff"> {{ activeIndex + 1 }} <span v-translate>of</span> {{ mediaFiles.length }} </span>
+          <span v-if="!$data.$_loader_folderLoading" class="uk-text-small" style="color:#fff"> {{ activeIndex + 1 }} <span v-translate>of</span> {{ mediaFiles.length }} </span>
```

Comment:
This should be accessible without `$data`

Reply:
It's not because it's prefixed with $ (would be the same for only _). Gave me a vue error that I needed to use $data.